code/modules/mob/mob_movement.dm

```diff
@@ -503,61 +504,22 @@
 ///Called by /client/Move()
 ///For moving in space
 ///Return 1 for movement 0 for none
-/mob/proc/Process_Spacemove(var/check_drift = 0,var/ignore_slip = 0)
+/mob/Process_Spacemove(var/check_drift = 0,var/ignore_slip = 0)
 	//First check to see if we can do things
 	if(restrained())
 		return 0
-
-	/*
-	if(istype(src,/mob/living/carbon))
-		if(src.l_hand && src.r_hand)
+	if(flying)
+		inertia_dir = 0
+		return 1
+
+	if(..())
+		//Check to see if we slipped
+		if(!ignore_slip && prob(Process_Spaceslipping(5)))
```

Comment:
Might be good to make this check for on_foot() to prevent people slipping inside vehicles and such